### DISCUSSION TITLE
建议加解密版块中尽量成对布局，比如sm4加密挨着sm4解密，sm2加密挨着sm2解密

### DISCUSSION THREAD

**Discussion by Asura88** [he3-app/he3-feedback#184] [General]
<img width="1440" alt="image" src="https://github.com/he3-app/he3-feedback/assets/25000885/291f370a-2141-4a24-9a08-4455b22d976e">
建议加解密版块中尽量成对布局，比如sm4加密挨着sm4解密，sm2加密挨着sm2解密

**Comment by lyzhang1999** [comment]
感谢你的建议。
目前你可以先拖动它们进行自定义排序来实现。
